samples/winapi_filefind/main.c

```diff
@@ -38,24 +39,29 @@ int main(void)
         } else {
             debugPrint("File     : ");
         }
-
         debugPrint("%s\n", findFileData.cFileName);
     } while (FindNextFile(hFind, &findFileData) != 0);
 
     debugPrint("\n");
 
-    DWORD error = GetLastError();
-    if (error == ERROR_NO_MORE_FILES) {
+    DWORD noMoreFilesError = GetLastError();
+    if (noMoreFilesError == ERROR_NO_MORE_FILES) {
         debugPrint("Done!\n");
     } else {
-        debugPrint("error: %x\n", error);
+        debugPrint("error: %x\n", noMoreFilesError);
     }
 
     FindClose(hFind);
 
+unmount_c:
+    ret = nxUnmountDrive('C');
+    // If there was an error while unmounting
+    if (!ret) {
+        DWORD unmountError = GetLastError();
```

Comment:
nit: `DWORD error` is fine, because there's no other variable to confuse this with, the scope is also tiny, so it's very clear what this is referring to.
Giving it an explicit name is just inviting copy/paste errors.

Reply:
There are many types of errors in the sample, like mountError/unmountError and fileFindError, each with a different scope, I wouldn't be so sure that it's not confusing.

Reply:
If they are in a different scope, then they don't need special name prefixes / suffixes.
We also don't call this `nxdkWinapiFileFindSampleMainUnmountError` (for example) because we just don't have to be that specific - longer names don't help with readability, but make it harder to adapt existing code for different situations.
Names in a narrow scope can (and should) have generic names in my opinion.